app/actions/index.js

```diff
@@ -191,3 +203,12 @@ export const dropFolder = folder => async dispatch => {
   if (!isDirectory) return
   addDat({ path: folder.path })(dispatch)
 }
+
+export const loadFromLocalStorage = () => dispatch => {
+  const blob = localStorage.getItem('state')
+  if (!blob) return
+  const dats = JSON.parse(blob)
+  for (const dat of dats) {
```

Comment:
adding to my former point: This should be called something other than dat  in order to have structure <-> name consistency. `datPath`?

Reply:
Hm,  I think `dat` is fine given `key` and `path` are properties on dat?

Reply:
`dat` is an object created with `dat-node` - no?

Reply:
I agree, renamed it to `datOpt`